e2e-tests/src/tests/scheduler.test.ts

```diff
@@ -272,9 +270,9 @@ test.describe('Scheduling', () => {
           peelOutputIsThere = true
           expect(topic.events.length).toEqual(12)
           break
-        case "/producer":
-          producerIsThere = true
-          expect(topic.events.length).toEqual(0)
+        default:
+          test.fail(topic.events.length !== 0, "Unexpected topic with events: "+topic.name)
```

Comment:
Is this change simply refactoring, or is it related to the changes to simulation templates made in this PR? If it's simply refactoring, could we pull it into its own commit?

Reply:
Done 👍